databuilder/databuilder/publisher/neo4j_csv_unwind_publisher.py

```diff
@@ -317,23 +319,38 @@ def _create_props_body(self,
         template = Template("""
             {% for k in record_keys %}
                 {% if preserve_empty_props %}
-                    {{ identifier }}.{{ k }} = row.{{ k }}
+                    {% if k in prop_types_to_configure %}
+                        {{ identifier }}.{{ k }} = {{ prop_types_to_configure[k] }}(row.{{ k }})
```

Comment:
if you were to convert something to another type like string you would have to do toString instead of just string right?

Reply:
That's correct, whichever cypher functions are available would need to be used as the associated value in the dict (toString being one option)